README.md

```diff
@@ -11,8 +11,8 @@ Create 3 promises:
 Add `success` and `error` handlers to each promise to show `<div data-qa="notification">` with `success` or `error` class and a promise message.
 
 ## Instructions
-1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_promises_practice_DOM/)
+1. Replace https://github.com/sinner1993/js_promises_practice_DOM with your Github username in the link
+    - [DEMO LINK](https://sinner1993.github.io/js_promises_practice_DOM/)
```

Comment:
You need to add the demo link to the PR description 
<img width="746" alt="image" src="https://github.com/user-attachments/assets/58232a76-52e5-40a2-9227-f0d0733ffce1">